test/ex_plasma/utxo_test.exs

```diff
@@ -4,6 +4,24 @@ defmodule ExPlasma.UtxoTest do
 
   alias ExPlasma.Utxo
 
+  describe "new/1" do
```

Comment:
Should we follow this style guide on assertions? https://github.com/lexmag/elixir-style-guide#exunit

> When asserting (or refuting) something with comparison operators (such as `==`, `<`, `>=`, and similar), put the expression being tested on the left-hand side of the operator and the value you're testing against on the right-hand side.